src/index.ts

```diff
@@ -4,11 +4,11 @@ import express from 'express';
 import serverless from 'serverless-http';
```

Comment:
Can we move the `dotenv.config()` line to just above the `global.fetch` line? It seems odd to have a function call above other imports. You are supposed to have the `dotenv.config()` [as early as possible](https://github.com/motdotla/dotenv#usage), but really that just means before any of *your* code. So putting it after external imports is fine.

Reply:
Actually, I think we can remove the import of `dotenv` and the `dotenv.config()` because we no longer read from a `.env` file.